src/unstract/sdk/adapters/vectordb/no_op_vectordb/src/no_op_vectordb.py

```diff
@@ -0,0 +1,32 @@
+import time
+
+from llama_index.core.schema import TextNode
+from llama_index.core.vector_stores.types import VectorStore, VectorStoreQueryResult
+
+
+class NoOpVectorDB(VectorStore):
+    stores_text: bool = True
+    stores_node: bool = True
+    is_embedding_query: bool = True
+    wait_time: int = 0
+
+    def __init__(
+        self,
+        wait_time: int,
+        dim: int,
+    ) -> None:
+        """Initialize params."""
+        wait_time = wait_time
+        dim = dim
+
+    def query(self, query) -> VectorStoreQueryResult:
+        """Query vector store."""
+        node1 = TextNode(text="This is a dummy document.", id_="1")
+        similarity_scores = [0.9]
+        embeddings = [["test"]]  # Dummy embeddings for each node
+
+        query_result = VectorStoreQueryResult(
+            nodes=[node1], similarities=similarity_scores, ids=embeddings
+        )
+        time.sleep(self.wait_time)
+        return query_result
```

Comment:
@harini-venkataraman Don't we need add() also here to dummify add?

Reply:
I do see that for simplicity we have skipped a chunk of flows. . Pls cross check once to make sure if we have dummified the required core functions from llama-index that are being used in our flows.

Reply:
@gaya3-zipstack In indexing flow, we have skipped the snippet where the actual addition of nodes happen. It does not trigger indexing and completes with an assumption that it is already indexed. 
While retrieval, the overridden `query` method caters to the dummy nodes. 
If needed, wee can get on a quick call to discuss this.

Reply:
Got it @harini-venkataraman